src/primitives.js

```diff
@@ -1092,25 +1096,132 @@ export default {
 			}
 		}
 
-		const polyhedrons = new InstancedMesh(
+		const offset = new Float32Array(coords.length * 3);
```

Comment:
What is the role of `offset` it has a somewhat generic name.

Reply:
It is the position offset of each polyhedron. The name "position" is used for the vertices of the triangles of the polyhedron geometry.

Reply:
I don't understand.  You mean the position of the center of a polyhedron? If that's tterhe case how about the name "center_position"?  And "surface_vertex" or "surface_point" for the other?

The outer faces of a polyhedron are sometimes triangles as in the case of tetrahedron, but sometimes not - the figure in cyan seems to be made out of pentagons.

Reply:
> And "surface_vertex" or "surface_point" for the other?

Let it "position" is the standard, to change that I'd need to use a `RawShaderMaterial` instead of `ShaderMaterial`, otherwise there'd be a useless line in the top of the GLSL code (three.js modify the GLSL code before compiling that).

> You mean the position of the center of a polyhedron? If that's the case how about the name "center_position"?

Yes, that is a better name.

> The outer faces of a polyhedron are sometimes triangles as in the case of tetrahedron, but sometimes not - the figure in cyan seems to be made out of pentagons.

Yes, but internally a pentagon is made of triangles.